### ISSUE TITLE
Bigger buttons for mobile 

### ISSUE THREAD

**Issue by danielo515** [roma0104/gsd5#27]
It's pretty hard to add task to a project while you are on a mobile phone. The plus sign is just not enough. A cool big button easy to tap will be better.  This can be applied to other ui elements

**Comment by roma0104** [comment]
@danielo515 Have you had issues with this on mobile devices since the CSS styling was applied to make the addition symbol larger?  I have tested it on a iPhone 4S, a larger Android phone, and a Galaxy Tab 4.  I have not had any trouble with selecting this feature but I am curious if you personally are having troubles.

**Comment by danielo515** [comment]
Hello @roma0104 

I did not tested the latest version. On my phone I'm using a version slightly modified. I'll check and let you know. Should I suppose I can try directly from the demo site?

**Comment by roma0104** [comment]
Yeah I think it is on the latest release but gsd5-nightly.tiddlyspot.com
for certian is updated.  Let me know.  I have toyed with the idea of
"mobile mode" but not sure if it is needed.  I just may tolerate small UI
better.

On Wed, Apr 8, 2015 at 3:08 PM, Daniel Rodríguez Rivero <
notifications@github.com> wrote:

> Hello @roma0104 <https://github.com/roma0104>
>
> I did not tested the latest version. On my phone I'm using a version
> slightly modified. I'll check and let you know. Should I suppose I can try
> directly from the demo site?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/roma0104/gsd5/issues/27#issuecomment-91020801>.
>



-- 
Roma Quintin Hicks

**Comment by danielo515** [comment]
Hello. It is definitely better now. It's easier to tap on the button, but I'm still not understanding why do you want to stick to such small buttons.

For the "mobile" mode my guess is that we should have a fixed  topbar, with just one button for adding a new task. Then ask the user to categorize the task (next, future, real, context...). This way we also avoid the need of opening the task for basic adjustment.

**Comment by danielo515** [comment]
I have created some kind of mobile theme. You can check it out if you want : mobiletw5.tiddlyspot.com

**Comment by danielo515** [comment]
Just to be clear: the advanced mode of task creation should be a tiddler. I hate TW modals
